source/Sale.h

```diff
@@ -18,6 +18,46 @@ PARTICULAR PURPOSE.  See the GNU General Public License for more details.
 
 #include <set>
 #include <string>
+#include <algorithm>
+
+
+template <class Item>
+class OutfitSale;
+
+
+
+// Container used for the price and selling type of different items (outfits in this case).
+class Sold {
+public:
+	enum ShowSold {
+		NONE = 0,
+		DEFAULT = (1 << 0),
```

Comment:
this way you can just do OutfitterSale()->GetShown(item) and check if it returns NONE
also by default if an element is not found in a map (like in const std::string &Sold::GetShow() const) it will return the last element
I hate that by default maps do that thing, I much prefer to have an indication that it wasn't found